README.md

```diff
@@ -33,3 +33,13 @@ The software will print an output to the console for each packet that is receive
 If the NCP fails to receive a valid frame with the timeout period set with the ```timeout``` command line parameter, then the NCP will be restarted. This will allow the sniffer to recover from serial port or NCP communications problems. The timer defaults to 30 seconds.
 
 A compiled JAR file can be found [here](https://www.cd-jackson.com/downloads/ZigBeeSniffer.jar) along with [further documentation](https://www.cd-jackson.com/downloads/ZigBeeWiresharkSniffer.pdf).
+
+When using Wireshark to display the packets, the raw IEEE 802.15.4 packet received by the Ember module is first encapsulated in a "TI CC24xx" frame format, then in a ZEPv2 (ZigBee Encapsulation Protocol version 2) frame format before being sent using UDP.
+Using the "TI CC24xx" frame format permit passing the RSSI value but has also limitations:
+* The "RSSI" value is correctly sent using a signed integer value in dBm
+* The "FCS Valid" field is always set to true as the Ember module discards invalid packets
+* The "LQI Correlation Value" is limited to a range of 0 to 127 (whereas the Ember module and the norm are defining this value for the range 0 to 255), so the displayed value is divided by 2.
+
+The real LQI value reported by the module in the range 0 to 255 should be displayed in the ZigBee Encapsulation Protocol section, but due to a bug, this isn't actually the case (see https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=16369).
```

Comment:
This is incorrect. You have fixed the bug here, so why say that the RSSI is not shown? Once this PR is merged, it will work so documenting this is invalid.

Reply:
These are differents things, there is information in the ZEP frame (the full LQI value) but it isn't displayed due to the dissector bug.

And there is the CC-24xx frame that also provide a LQI value but limited to 7 bits. So there are two LQI values provided. RSSI is now correctly displayed with the fix provided in the CC-24xx frame (there is no RSSI field in the ZEP frame).

Documentation is valid for me.

Reply:
Ok, thanks, I missed that.